src/app/view/components/dot-theme-selector-dropdown/dot-theme-selector-dropdown.component.ts

```diff
@@ -48,6 +49,10 @@ export class DotThemeSelectorDropdownComponent
             .getCurrentSite()
             .pipe(
                 pluck('identifier'),
+                tap((identifier) => {
+                    this.currentSiteIdentifier = identifier;
+                    return identifier;
+                }),
                 mergeMap((identifier: string) => {
```

Comment:
you can set `currentSiteIdentifier ` inside the `mergeMap`. no need for the `tap`

Reply:
This was done on purpose @hmoreras, side effects must be done in `tap` in my opinion. Happy to discuss.